packages/sanity/src/core/store/events/getDocumentChanges.ts

```diff
@@ -0,0 +1,261 @@
+import {diffInput, wrap} from '@sanity/diff'
+import {type SanityDocument, type TransactionLogEventWithEffects} from '@sanity/types'
+import {applyPatch, incremental} from 'mendoza'
+import {
+  combineLatest,
+  from,
+  map,
+  type Observable,
+  of,
+  shareReplay,
+  startWith,
+  switchMap,
+  tap,
+} from 'rxjs'
+import {type SanityClient} from 'sanity'
+
+import {type Annotation, type ObjectDiff} from '../../field'
+import {wrapValue} from '../_legacy/history/history/diffValue'
+import {getDocumentTransactions} from './getDocumentTransactions'
+import {
+  type DocumentGroupEvent,
+  type EventsStoreRevision,
+  isCreateDocumentVersionEvent,
+} from './types'
+import {type EventsObservableValue} from './useEventsStore'
+
+const buildDocumentForDiffInput = (document?: Partial<SanityDocument> | null) => {
```

Comment:
Huh, don't we already have something like this in the studio? 🤔 
Feels odd that we would have a way of presenting this information already